src/load/azext_load/tests/latest/resources/config-disable-public-ip-false.yaml

```diff
@@ -0,0 +1,15 @@
+displayName: CLI-Test
+testPlan: sample-JMX-file.jmx
+description: 'Test created from az load test command'
+engineInstances: 1
+failureCriteria:
+  - avg(requests_per_sec) > 78
+  - percentage(error) > 50
+  - GetCustomerDetails: avg(latency) > 200
+env:
+  - name: 'rps'
+    value: 1
+publicIPDisabled: False
```

Comment:
Shouldnt this be `publicIPDisabled: false` ?

Reply:
We should have one yaml with true as well preferrably

Reply:
added for true case as well